CONTRIBUTING.md

```diff
@@ -112,7 +112,7 @@ Datasets should take no arguments and return an AnnData object. If `test is True
 function dataset(bool test=False) -> AnnData adata
 ```
 
-Methods should take an AnnData object and store the output in `adata.obs` according to the specification of the task. If `test is True`, you may modify hyperparameters (e.g. number of iterations) to make the method run faster.
+Methods should take an AnnData object and store a) the output in `adata.obs` according to the specification of the task and b) the version of the package used to run the method in `adata.uns["method_code_version"]`. If `test is True`, you may modify hyperparameters (e.g. number of iterations) to make the method run faster.
```

Comment:
I'm not sure a) is true anymore... some methods have outputs not in .obs but e.g., .obms (batch integration, and i assume maybe also multimodal data integration). I like the method_code_version part. We'd need to adapt that in all methods then though.

Reply:
You're right, a) is now more nuanced. I can change that to reflect the reality. 

Re: adapting in all methods, I've done so for most/all that are currently on main. The backports will allow the old system for the time being so as to not break any existing PRs.